cmd/poller/plugin/label_agent/parse_rules.go

```diff
@@ -491,9 +493,14 @@ type valueToNumRule struct {
 
 func (me *LabelAgent) parseValueToNumRule(rule string) {
 	if fields := strings.Fields(rule); len(fields) == 4 || len(fields) == 5 {
-		r := valueToNumRule{metric: fields[0], label: fields[1]}
+		r := valueToNumRule{metric: fields[0], label: fields[1], isRegex: false}
```

Comment:
we should discuss this more tomorrow. This isn't regex support, more like simple wildcard. Probably better to add value_to_num_regex instead of slowly accreting regex like features here

Reply:
Yes, Agreed.

Reply:
Done